yaml-tests/src/test/resources/inserts-updates-deletes.yamsql

```diff
@@ -109,10 +109,19 @@ test_block:
       # Case where not all values are provided of A. Still works, since the columns for which no values are provided can be nullable.
       - query: insert into A(A1, A2, A3) values (4);
       - supported_version: !current_version
-      - count: 1
+      - error: "42601"
     -
       # Case when the number of values is more than the number of columns specified.
       - query: insert into A(A1, A2, A3) values (5, 6, 7, 8, 9);
       - supported_version: !current_version
       - error: "42601"
+    -
+      # Case when a nullable column's value is not provided
```

Comment:
Just to cover that, could you add a test where the not provided column is the first one and another case where it's the last one?

Reply:
done!